### PR TITLE
Fenix probes load by URL

### LINKED ISSUE #747: Support product-specific probe details
The *probe details* view should support product-specific configurations and content. We should show different content for Fenix probes than we do for Firefox probes, for example.

Hamilton sketched out how he suggested this be implemented:

* Add a *DetailsContainer* component which doesn't hard-code Firefox-related content (e.g., documentation)
* Implement a *FirefoxDetails* component
* Implement a *FenixDetails* component

### LINKED ISSUE #748: Reduce the cost of adding new products in terms of routing
For example, conditionals like this are not great:

```javascript
if (product === 'firefox') {
  // do this
} else if (product === 'fenix') {
  // do that
}
```

Hamilton has a pattern for addressing this kind of problem in GLAM:
https://github.com/mozilla/gud/blob/master/src/routes/routes.js

### PR DESCRIPTION
**Pull request by hamilton**
closes #747, closes #748

This PR does the remaining small bits to technically get Fenix probes to land in GLAM. It (1) creates a `routes.js` file that allows for probe detail views per product, (2) creates an empty fenix detail view while providing the current firefox one (this is not really how we should move forward but it paves the way).

It turns out this was the last bit that was needed to unlock probes in a generalized way on pageload. Search does _not_ work for probes as of this PR; blocked by the PSS work.

### CODE REVIEW

File: src/components/controls/PercentileSelectionControl.svelte [hunk 1]
```diff
@@ -9,9 +9,25 @@
     return `${v}% of clients have seen this value or one smaller`;
   }
 </script>
-<BodyControl options={[{ label: '5%', value: 5, labelColor: cmp(5), tooltip:
-t(5), }, { label: '25%', value: 25, labelColor: cmp(25), tooltip: t(25), }, {
-label: '50%', value: 50, labelColor: cmp(50), tooltip: t(50), }, { label: '75%',
-value: 75, labelColor: cmp(75), tooltip: t(75), }, { label: '95%', value: 95,
-labelColor: cmp(95), tooltip: t(95), }]} reverse={true}
+<BodyControl options={[{
```
Comment (hamilton) [inline src/components/controls/PercentileSelectionControl.svelte:12]: removing linting errors.

File: src/components/controls/ProbeViewControl.svelte [hunk 2]
```diff
@@ -8,7 +8,6 @@ import { store, currentQuery } from '../../state/store';
 
 import BodyControl from './BodyControl.svelte';
 
-export let selections;
```
Comment (hamilton) [inline src/components/controls/ProbeViewControl.svelte:11]: more linting error removal.

**Comment by openjck** [comment]
Woah, this brings ESLint problems down by **a lot**. Would you mind lowering the max to 25 in *scripts/eslint-health*?

File: src/routing/pages/probe/FirefoxLegacyDetails.svelte [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+<div class="probe-details-download">
+
+  <div class="drawer-section drawer-section--end">
+    <!-- FIXME: once @graph-paper/button supports href, use that instead. -->
+    <!-- <button
+        on:click={exportData}
+        class="docs-button"
+      >
+        <Brackets size={16} /> Export to JSON
+      </button> -->
```
Comment (openjck) [inline src/routing/pages/probe/FirefoxLegacyDetails.svelte]: Commented code

File: src/state/store.js [hunk 4]
```diff
@@ -79,61 +80,48 @@ store.reset = () => {
...
-  store.setDimension(
-    'aggregationLevel',
-    getDefaultFieldValue('aggregationLevel')
-  );
-  store.setDimension('process', getDefaultFieldValue('process'));
+store.resetProductDimensions = () => {
+  const config = getActiveProductConfig();
+  Object.entries(config.dimensions).forEach(([key, { defaultValue }]) => {
+    store.setDimension(key, defaultValue);
+  });
```
Comment (openjck) [inline src/state/store.js:124]: :+1: Thanks!

File: src/state/store.js [hunk 5]
```diff
@@ -79,61 +80,48 @@ store.reset = () => {
   });
 };
 
-function getActiveProductConfig() {
+export function getActiveProductConfig() {
```
Comment (openjck) [inline src/state/store.js]: It doesn't look like this is used externally. Is the plan to use it externally in the future?

File: src/config/fenix.js [hunk 6]
```diff
@@ -92,8 +96,18 @@ export default {
           aggregationLevel,
           metricType
         );
+        // delete old (bad) builds?
```
Comment (openjck) [inline src/config/fenix.js:99]: I'm not sure if I understand what this comment means. Why the question mark?

**Comment by openjck** [comment]
Overall, I like this. No real complaints from me. This matches what I expected based on what you had told me you were thinking about.

File: src/routing/pages/probe/FirefoxLegacyDetails.svelte [hunk 7]
```diff
@@ -0,0 +1,236 @@
...
+
+  .docs-button:hover {
+    background-color: var(--cool-gray-150);
+  }
+</style>
+
+<!-- probe-details-content -->
+<div class="probe-details-content">
+  <div class="probe-details-overview">
+    {#if $probe.type}
```
Comment (openjck) [inline src/routing/pages/probe/FirefoxLegacyDetails.svelte]: > Uncaught (in promise) TypeError: can't access property "type", ctx[0] is undefined

## Steps to reproduce

1. Visit http://localhost:5000/firefox/probe/gc_ms/explore
2. Click on the GLAM logo to go home

File: src/state/store.js (hunk 5 above)
Comment (hamilton) [reply src/state/store.js]: ah, I think I was going to use this somewhere, but decided against it. Will remove

File: src/config/fenix.js [hunk 8]
```diff
@@ -35,6 +38,7 @@ export default {
         { key: 'build_id', label: 'Build ID' },
         { key: 'version', label: 'Major Version' },
       ],
+      defaultValue: 'build_id',
     },
   },
```
Comment (robhudson) [inline src/config/fenix.js:43]: The backend now requires `app_id` be provided. It should be one of:
* `org_mozilla_fenix` for Fenix Nightly
* `org_mozilla_firefox_beta` for Fenix Beta
* `org_mozilla_firefox` for Fenix Release

**Pull request merged**